config/Migrations/20190814000000_AlterTemplateToEmailQueue.php

```diff
@@ -0,0 +1,40 @@
+<?php
+use Migrations\AbstractMigration;
+use Phinx\Db\Adapter\MysqlAdapter;
+
+class AlterTemplateToEmailQueue extends AbstractMigration
+{
+    /**
+     * Up Method
+     *
+     * More information on this method is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-up-method
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $this->table('email_queue')
+            ->changeColumn('template', 'string', [
+                'limit' => 255,
```

Comment:
what's the actual longest template name that you have?

Reply:
@lorenzo  For example: _Passbolt/MultiFactorAuthentication.LU/mfa_user_settings_reset_
It is 61 chars long, but potentially, it could be longer.

Reply:
let's make it 100, otherwise it is not indexable by mysql, in case we need to index it

Reply:
Good point. With utf8mb4 collation, it would indeed fail.
"100" seems enough for a template path, and we still have room for more if needed (can go up until 191).

Should we throw an exception if larger or do we let the driver handle it?

Reply:
sounds like a good idea

Reply:
its done